src/containers/shared/components/Transaction/EscrowFinish/Description.tsx

```diff
@@ -25,30 +29,31 @@ const Description: TransactionDescriptionComponent = (
       <div>
         {t('escrow_completion_desc')} <Account account={data.tx.Account} />
       </div>
-      <div>
+      <div data-testid="amount-line">
         <Trans i18nKey="escrow_completion_desc_2">
           The escrowed amount of
           <b>
-            {normalizeAmount(deleted.FinalFields.Amount, language)}
-            <small>XRP</small>
+            <Amount value={formatAmount(deleted.FinalFields.Amount)} />
           </b>
           was delivered to
           <Account account={deleted.FinalFields.Destination} />
         </Trans>
-        {deleted.FinalFields.Destination === data.tx.Account && (
-          <span>
-            {' '}
-            (
-            <b>
-              {normalizeAmount(
-                deleted.FinalFields.Amount - parseInt(data.tx.Fee || '0', 10),
-                language,
-              )}
-              <small>XRP</small>
-            </b>{' '}
-            {t('escrow_after_transaction_cost')})
-          </span>
-        )}
+        {isXRP(deleted.FinalFields.Amount) &&
+          deleted.FinalFields.Destination === data.tx.Account && (
+            <span>
+              {' '}
+              (
+              <b>
+                <Amount
+                  value={formatAmount(
+                    deleted.FinalFields.Amount -
+                      parseInt(data.tx.Fee || '0', 10),
+                  )}
+                />
+              </b>{' '}
+              {t('escrow_after_transaction_cost')})
+            </span>
```

Comment:
```suggestion
            <div>
              (
              <b>
                <Amount
                  value={formatAmount(
                    deleted.FinalFields.Amount -
                      parseInt(data.tx.Fee || '0', 10),
                  )}
                />
              </b>{' '}
              {t('escrow_after_transaction_cost')})
            </div>
```
Using a `div` will automatically create a new line for you.